src/components/date/Time.tsx

```diff
@@ -12,5 +12,5 @@ import { format, parseISO } from "date-fns";
  */
 export const Time = ({ time }: { time: Date | string }) => {
   const date = typeof time === "string" ? parseISO(time) : time;
-  return <>{format(date, "HH:mm")} UTC</>;
+  return <>{format(date, "HH:mm")}</>;
```

Comment:
I think this should still display the current timezone. This is easier to do with React Context (see below).